master/buildbot/test/unit/test_process_properties.py

```diff
@@ -64,6 +65,24 @@ def asDict(self):
         }
         return ds
 
+    def asSSDict(self):
+        return self.asDict()
+
+
+class FakeChange:
```

Comment:
I don't think it makes much sense to use FakeChange/FakeSource. TempChange/SourceStamp is nearly the same, and can be used as is. So that it is also tested by this test.
I believe this code predates TempSourceStamp, so this is why it is weirdly implemented.